internal/tools/orchestrator/hepa/providers/domain/provider.go

```diff
@@ -60,17 +71,67 @@ func (p *provider) Init(ctx servicehub.Context) error {
 	}
 	if p.Register != nil {
 		type domainService = pb.DomainServiceServer
-		pb.RegisterDomainServiceImp(p.Register, p.domainService, apis.Options(), p.Perm.Check(
+		pb.RegisterDomainServiceImp(p.Register, p.domainService, apis.Options(), p.getCheckParam(), p.Perm.Check(
 			perm.Method(domainService.GetOrgDomains, perm.ScopeOrg, "org", perm.ActionGet, perm.OrgIDValue()),
```

Comment:
Verify whether there are any permission leaks in other existing interfaces as well.